main.py

```diff
@@ -93,24 +180,31 @@ def should_nag(message: discord.Message) -> bool:
     3. Embed is safe and there is a video
```

Comment:
Docstring should probably be updated to reflect the more complex behavior

Reply:
Updated to match